homeassistant/components/environment_canada/sensor.py

```diff
@@ -297,13 +297,14 @@ def native_value(self):
 class ECSensor(ECBaseSensor):
     """Environment Canada sensor for conditions."""
 
-    def __init__(self, coordinator, description):
+    def __init__(self, coordinator, description, enabled_default=True):
```

Comment:
You don't need to set that here, instead, set it on the `AQHI_SENSOR` entity description.

Reply:
We are testing out another approach where we don't need to disable this sensor by default. I put this to draft while we test out that approach.